### PR TITLE
Removed unnecessary lines, changed silicon laws, narrowed map selection.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -7,127 +7,4 @@
...
-Cyberboss = Game Master
-PJB3005 = Game Master
-Sweaterkittens = Game Master
-Feemjmeem = Game Master
-JStheguy = Game Master
-excessiveuseofcobby = Game Master
-Plizzard = Game Master
-octareenroon91 = Game Master
\ No newline at end of file
+towershield = Host
```
Comment (TrustyGun) [inline config/admins.txt]: uh

Comment (Towershield) [inline config/admins.txt]: A screw up on my end, sent a change to the wrong place.  Please disregard.

File: config/config.txt [hunk 2]
```diff
@@ -121,7 +121,7 @@ NORESPAWN
 # DONT_DEL_NEWMOB
 
 ## set a hosted by name for unix platforms
-HOSTEDBY Yournamehere
+HOSTEDBY Towershield
```
Comment (Anonmare) [inline config/config.txt]: Another one here

File: config/silicon_laws.txt [hunk 3]
```diff
@@ -3,6 +3,7 @@
 #Empty lines and lines starting with # are ignored.
 #~Miauw
 
+You will not treat Lizardmen, Flies, Plasmamen, any variety of Undead and their thralls and members of the crew who are confirmed members of any cults, as human beings.
```
Comment (Anonmare) [inline config/silicon_laws.txt]: Controversial change here. Mind controlled and Cultist Humans being valid salad by default under Asimov will cause contention

Comment (Towershield) [inline config/silicon_laws.txt]: Change to the wrong place. Please disregard.

**Comment by ShizCalev** [comment]
mems

File: config/config.txt [hunk 4]
```diff
@@ -1,17 +1,17 @@
 ## Server name: This appears at the top of the screen in-game and in the BYOND hub. Uncomment and replace 'tgstation' with the name of your choice.
-# SERVERNAME tgstation
+ SERVERNAME Omega_Outpost
```
Comment (MoreRobustThanYou) [inline config/config.txt]: srsly

**Comment by optimumtact** [comment]
you bullied them so hard they deleted their account

**Pull request closed without merging**